README.md

```diff
@@ -63,6 +63,8 @@ The AWS AppSync SDK for iOS enables you to access your AWS AppSync backend and p
     * AWSCore.framework
     * Reachability.framework
     * SQLite.framework
+    * AppSyncRealTimeClient.framework
+    * Starscream.framework
```

Comment:
Below, at https://github.com/awslabs/aws-mobile-appsync-sdk-ios/pull/367/files#diff-04c6e90faac2675aa89e2176d2eec7d8R86

Change "using Xcode 10.1" to "using Xcode 11". We should also add a new item to update the image to `osx_image: xcode11.3` in .travis.yml, and then update README.md as appropriate.

Reply:
thanks! updating README for now: https://github.com/awslabs/aws-mobile-appsync-sdk-ios/pull/369
and travis image to latest https://github.com/awslabs/aws-mobile-appsync-sdk-ios/pull/370